test/E2E/Helpers.purs

```diff
@@ -0,0 +1,382 @@
+module Test.E2E.Helpers
```

Comment:
Module name should match directory hierarchy, regardless of where the module is located (`src` or `test`). PS does not enforce this, but we should.

Reply:
But it does, no?

Reply:
It should be `test/Test/E2E/Helpers.purs`, because the first directory is a source directory (see `spago.dhall`, so it does not count).

Reply:
I think it's better to adjust the module name and not the filepath

Reply:
@firefrorefiddle you are right. Then leave it as is, I will fix it for the whole project.

Reply:
@klntsky I can do it later if you like. But it will mean adjusting the filepath rather than the module name because the `Test.`-prefix is what prevents name clashes with the tested modules.